utils.js

```diff
@@ -7,7 +7,13 @@ const getSightings = async () => {
   return JSON.parse(sightings);
 };
 
+async function getSightingsIndexSelected(sightingsIndex) {
```

Comment:
i would call this `getSighting` or `getOne` instead. Singular should suffice usually